### PR TITLE
Adjusted combined given-when steps in Logging context

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
`Given-When` steps splitted into seperate functions and `check-success` assertion added on given steps

## Related Issue
- Part of https://github.com/owncloud/QA/issues/635

### CODE REVIEW

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/core/22212/47/12
@kiranparajuli589 needs rebase to current core master

**Review by dpakach — COMMENTED** [review]
LGMT :+1:

File: tests/acceptance/features/bootstrap/Logging.php [hunk 1]
```diff
@@ -413,10 +448,29 @@ public function owncloudLogTimezoneIsSetTo($timezone) {
...
+		$currentTimezone = LoggingHelper::getLogTimezone();
+		Assert::assertEquals(
+			$currentTimezone,
+			$timezone
+		);
+	}
+
 	/**
 	 * @When the owncloud log is cleared
 	 * @Given the owncloud log has been cleared
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/Logging.php]: This still needs to be separated - is there a reason that it has not been separated?

**Review by phil-davis — APPROVED** [review]

File: tests/acceptance/features/bootstrap/Logging.php (hunk 1 above)
Comment (phil-davis) [reply tests/acceptance/features/bootstrap/Logging.php]: @kiranparajuli589 if it makes sense to separate `Given` here, then do it in another PR

Comment (kiranparajuli589) [reply tests/acceptance/features/bootstrap/Logging.php]: OK @phil-davis!
I might have forgetten to split them in hurry to split others :smiley: :scissors:

**Pull request merged**